src/languages/r.js

```diff
@@ -118,7 +126,7 @@ export default function(hljs) {
       hljs.HASH_COMMENT_MODE,
 
       {
-        className: 'string',
+        scope: 'string',
```

Comment:
This noise makes the PR much harder to review when it's now separated in it's own commit.

Reply:
Gah, true. I can pull it out if you want.

Reply:
Didn't turn out to be so terrible, but thought for the future...